test/Microsoft.Health.Dicom.Tests.Integration/Features/TranscoderTests.cs

```diff
@@ -152,7 +152,7 @@ private string GetFramesHashCode(DicomFile dicomFile)
 
         private string GetByteArrayHashCode(byte[] byteArray)
         {
-            return Convert.ToBase64String(new SHA1Managed().ComputeHash(byteArray));
+            return Convert.ToBase64String(HashAlgorithm.Create(nameof(SHA1)).ComputeHash(byteArray));
```

Comment:
We need to use something like SHA256.

Reply:
Our tests files make the assumption we use SHA1, so I'll make this change in another PR